packages/TBC721/test/nft.test.ts

```diff
@@ -37,62 +37,159 @@ describe('NFT', () => {
+        expect(initialRoot).eq(nft._root)
+      })
+  
+      it("The _owners are updated to receiver's public key", () => {
+        expect(nft._owners).deep.eq([receiver.getPublicKey()])
+      })
+  
+      it("Syncing to the NFT's revision returns an object that is equal to the NFT", async () => {
+        expect(await receiver.sync(nft._rev)).deep.eq(nft)
+      })
+    })
   })
 
-  describe('Constructor', () => {
-    it('Sender mints an NFT', async () => {
-      nft = await sender.new(NFT, ['Test'])
-      // @ts-ignore
-      expect(nft).matchPattern({ name: 'Test', symbol, ...meta })
+  describe('Using NFTs with the TBC721 helper class', () => {
+    const computer = new Computer(RLTC)
+    
+    let tbc721
+    let nft
+    
+    before(async () => {
+      await computer.faucet(1e7)
     })
-
-    it('Property _owners is a singleton array with minters public key', () => {
-      expect(nft._owners).deep.eq([sender.getPublicKey()])
+    
+    describe('Constructor', () => {
+      it('Should create a new TBC721 object', async () => {
+        tbc721 = new TBC721(computer)
+        expect(tbc721.computer).deep.eq(computer)
+      })
     })
-
-    it('Properties _id, _rev, and _root have the same value', () => {
-      expect(nft._id).eq(nft._rev).eq(nft._root)
-
-      initialId = nft._id
-      initialRev = nft._rev
-      initialRoot = nft._root
+  
+    describe('deploy', () => {
+      it('Should deploy a contract', async () => {
+        await tbc721.deploy()
+      })
     })
-
-    it("The nft is returned when syncing against it's revision", async () => {
-      expect(await sender.sync(nft._rev)).deep.eq(nft)
+  
+    describe('mint', () => {
+      it('Should mint an NFT', async () => {
+        nft = await tbc721.mint('name', 'symbol')
+      })
+    })
+  
+    describe('balanceOf', () => {
+      it('Should return the balance', async () => {
+        const balance = await tbc721.balanceOf(computer.getPublicKey())
+        expect(balance).to.be.greaterThanOrEqual(1)
+      })
+    })
+  
+    describe('ownerOf', () => {
+      it('Should return the owner', async () => {
+        const owners = await tbc721.ownersOf(nft._id)
+        expect(owners).deep.eq([computer.getPublicKey()])
+      })
+    })
+  
+    describe('transfer', () => {
+      it('Should transfer an NFT', async () => {
+        const computer2 = new Computer()
+        const nft2 = await tbc721.mint('name', 'symbol')
+        const publicKey2 = computer2.getPublicKey()
+        await tbc721.transfer(nft2._id, publicKey2)
+        const res = await tbc721.balanceOf(computer.getPublicKey())
+        expect(res).to.be.greaterThanOrEqual(1)
+      })
     })
   })
 
-  describe('transfer', async () => {
-    it('Sender transfers the NFT to receiver', async () => {
-      await nft.transfer(receiver.getPublicKey())
-      // @ts-ignore
-      expect(nft).to.matchPattern({ name: 'Test', symbol, ...meta })
-    })
+  describe.only('Examples from docs', () => {
```

Comment:
Remove this .only

Reply:
Great thanks, this was hiding an actual bug in the test. Fixed in [8e1b560](https://github.com/bitcoin-computer/monorepo/pull/173/commits/8e1b560ec6d8ac73b656dadbfd6ada006657d343).